MechJeb2/MechJebModuleNodeExecutor.cs

```diff
@@ -137,7 +156,7 @@ public override void OnFixedUpdate()
                 {
                     core.warp.WarpToUT(node.UT - halfBurnTime - leadTime);
                 }
-                else if (!MuUtils.PhysicsRunning() && core.attitude.attitudeAngleFromTarget() > 10 && timeToNode < 600)
+                else if (!MuUtils.PhysicsRunning() && core.attitude.attitudeAngleFromTarget() > 10 && timeToNode < halfBurnTime + 30)
```

Comment:
that needs to remain 600 seconds or something large like that.  if you don't you can wind up flipped around 180 degrees  30 seconds before the burn possibly with a huge vessel that cannot slew around that fast, and this will affect all users (stock, RSS/RO, principia, etc).

Reply:
How about make a conditional statement: 
```suggestion
                else if (!MuUtils.PhysicsRunning() && core.attitude.attitudeAngleFromTarget() > 90 && timeToNode < halfBurnTime + 120 || !MuUtils.PhysicsRunning() && core.attitude.attitudeAngleFromTarget() > 10 && timeToNode < halfBurnTime + 30)
```

Or it could be limited to Principia only:
```suggestion
                else if (!MuUtils.PhysicsRunning() && core.attitude.attitudeAngleFromTarget() > 10 && timeToNode < 600 || MechJebModuleGuidanceController.isLoadedPrincipia && !MuUtils.PhysicsRunning() && core.attitude.attitudeAngleFromTarget() > 10 && timeToNode < halfBurnTime + 30)```

Reply:
I'm a little confused about why you need to change this.  I thought this was the code where if you are very far from the time of the burn, MJ warps not caring at all about orientation (because stock KSP likes be wonky and cause some kind of precession even though its physics model shouldn't precess AFAIK -- but Principia probably precesses correctly) and then once you hit 10 minutes it starts to care about orientation and will drop you out of warp and fix the orientation if it gets too large.  That should AFAIK still apply to Principia?  Or maybe I've just forgotten what my own code patch does...

Reply:
The problem is that with Principia's node would have some kind of precession and causes node's direction move on the navball so that within 10 minutes from burn time it will drop out of warp like every 20 seconds because the node's direction has moved pretty far. It also happens when the node's reference frame is different than the navball's reference frame and that's even worse. It could be annoying if the ship's electric charge are low and in the shadow of the planet.

Reply:
Uh, you need to check a box in Principia.  Inertially fixed or something like that?

Reply:
Yeah, I’m aware of that. However, inertially fixed uses slightly more deltaV and not works if the navball’s reference frame is different than the node’s reference frame, which is sometimes used when planning complex trajectories.

Reply:
can you throw some kind of comment to that effect, specifically that it is to work around cases where the "inertially fixed" checkbox is insufficient?  since i wasn't aware those use cases existed.

Reply:
(oh and go with the limit-to-principia only case probably, keep the use cases totally separate.

Reply:
Uh, you mean like open a new issue?